DEWNR/developer-challenge
# Create Challenge website base structure - Mina Kim

Hi,

I had to spend unexpected time to set up Webpack and fix some issues for it.
As the result, I haven't completed the task fully.

**Description**
Make an one-page website using layout.jpg

**Steps to Test (production version)**
Please upload `challenge-response/dist` folder to web sever(static).
The `dist` folder needs to be the root of the site.

**Steps to Test (local)**
Download code and open `challenge-response` folder in the bash.
Run `npm install`
Run `npm start`
Check result
